addons/website_slides/tests/test_slide_utils.py

```diff
@@ -134,6 +135,35 @@ def test_mail_completed_with_different_templates(self):
             ['Congratulation! You completed %s' % self.channel.name, 'ATestSubject']
         )
 
+    def test_default_completion_time(self):
+        """Verify whether the system calculates the completion time when it is not specified,
+        but if the user does provide a completion time, the default time should not be applied."""
+
+        file_path = get_module_resource('base', 'tests', 'minimal.pdf')
```

Comment:
is there any way to test it without using _get_completion_time_pdf ?

ci/security is not happy :/

Reply:
i will check it and let you know 
:slightly_smiling_face:

Reply:
thanks :)

Reply:
Hello @std-odoo 

can i write the static time in the asset 
`        self.assertEqual(0.0833, slides.completion_time)
`

Reply:
ok for me :)

Reply:
Hello @std-odoo 

Changes are done.
Can you please check.

Thanks :slightly_smiling_face:

Reply:
you still use the open function which male ci/security not happy :/

Reply:
Hello @std-odoo 

I've observed that many test cases involve uploading a document, and I've tried using the Path function. However, the security checks in Runbot are still failing. Are there any alternative methods or approaches available to read the content of the file that may help resolve this issue?

Thanks :slightly_smiling_face:

Reply:
just don't use a pdf